packages/office-ui-fabric-react/src/components/Persona/PersonaCoin.styles.ts

```diff
@@ -0,0 +1,244 @@
+import {
+  IPersonaCoinStyleProps,
+  IPersonaCoinStyles,
+  PersonaSize,
+} from './Persona.types';
+import {
+  HighContrastSelector,
+  FontSizes,
+  FontWeights,
+} from '../../Styling';
+
+export const getStyles = (
+  props: IPersonaCoinStyleProps
+): IPersonaCoinStyles => {
+  const {
+    className,
+    size,
+    theme,
+  } = props;
+
+  const { palette } = theme;
+
+  // Persona size conditionals
+  const isSize10 = size === PersonaSize.size10 || size === PersonaSize.tiny;
+  const isSize16 = size === PersonaSize.size16;
+  const isSize24 = size === PersonaSize.size24 || size === PersonaSize.extraExtraSmall;
+  const isSize28 = size === PersonaSize.size28 || size === PersonaSize.extraSmall;
+  const isSize32 = size === PersonaSize.size32;
+  const isSize40 = size === PersonaSize.size40 || size === PersonaSize.small;
+  const isSize48 = size === PersonaSize.size48;
+  const isSize72 = size === PersonaSize.size72 || size === PersonaSize.large;
+  const isSize100 = size === PersonaSize.size100 || size === PersonaSize.extraLarge;
```

Comment:
These conditionals are repeated 3 times, we should extract them out to a common place to reduce duplication

Reply:
This one, however, is a bit more complicated. I'd have to make it an exported function that takes size as an argument then returns the correct boolean, right?